sample-apps/celery_sample/settings.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import os
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+class BaseConfig(object):
+    BASE_DIR = Path(__file__).parent.resolve()
+    # SECURITY WARNING: don't run with debug turned on in production!
+    DEBUG = True
+
+    SECRET_KEY = os.getenv("SECRET_KEY", "")
+
+    JUDOSCALE = {
+        "API_BASE_URL": "https://judoscale-celery.requestcatcher.com",
+        "LOG_LEVEL": "DEBUG",
+        "REPORT_INTERVAL_SECONDS": 2,
```

Comment:
```suggestion
        "REPORT_INTERVAL_SECONDS": 15,
```

I regret using a 2-second interval when I initially created the Django sample app. It makes so much noise when attempting to test! Let's start using longer intervals for sample apps so it's easier to see what's happening.